public/static/js/projects.js

```diff
@@ -1,1221 +1,1414 @@
-mciModule.controller('ProjectCtrl', function ($scope, $window, $http, $location, $mdDialog, timeUtil) {
-
-  $scope.availableTriggers = $window.availableTriggers
-  $scope.userId = $window.user.Id;
-  $scope.isAdmin = $window.isAdmin;
-  $scope.create = $window.canCreate;
-  $scope.validDefaultLoggers = $window.validDefaultLoggers;
-
-  $scope.projectVars = {};
-  $scope.patchVariants = [];
-  $scope.projectRef = {};
-  $scope.displayName = "";
-
-  $scope.projectView = false;
+mciModule.controller(
+  "ProjectCtrl",
```

Comment:
Most of these changes are automatic lint fixes that happened on save automatically. The main changes are: 
284 addGithubTrigger
291 removeGithubTrigger 
and line 460